packages/openapi-framework/index.ts

```diff
@@ -258,14 +258,10 @@ export default class OpenAPIFramework implements IOpenAPIFramework {
               const operationId = methodDoc.operationId;
               if (operationId && operationId in this.operations) {
                 const operation = this.operations[operationId];
-
                 acc[METHOD_ALIASES[method]] = (() => {
                   const innerFunction: any = operation;
                   innerFunction.apiDoc = methodDoc;
-                  // Operations get dependencies injected in `this`
-                  return innerFunction.bind({
-                    dependencies: { ...this.dependencies },
-                  });
```

Comment:
It _seems_ like this is  safe to remove, since [we inject dependencies later](https://github.com/kogosoftwarellc/open-api/blob/13ad704b645055d9baec6e047a2d1a1288edb538/packages/openapi-framework/index.ts#L323) on for all routes